GANDLF/losses/regression.py

```diff
@@ -7,16 +7,12 @@ def CEL(out, target, params):
     if len(target.shape) > 1 and target.shape[-1] == 1:
         target = torch.squeeze(target, -1)
 
-    class_weights = None
-    if params["class_weights"]:
-        class_weights = torch.FloatTensor(list(params["class_weights"].values()))
+    weights = None
+    if params["weights"]:
+        weights = torch.FloatTensor(list(params["weights"].values()))
+        weights = weights.float().to(target.device)
```

Comment:
This looks like it will break things, because what if params["weights"] is not given but params["class_weights"] is given?